langchain4j-google-ai-gemini/src/main/java/dev/langchain4j/model/googleai/Json.java

```diff
@@ -41,4 +42,12 @@ static <T> T fromJson(String json, Class<T> type) {
             throw new RuntimeException(e);
         }
     }
+
+    static <T, U> U convertValue(T fromValue, Class<U> toValue) {
```

Comment:
```suggestion
    static <T> T convertValue(Object fromValue, Class<T> toValue) {
```
Second type does not seems to be required here, as well as below

Reply:
Removed unused method and types. thanks